devel/apache-arrow/Portfile

```diff
@@ -24,17 +24,19 @@ checksums           rmd160  e2dbd95f5c9b5739a7351dd920b40509c4e968ca \
                     sha256  94cca341bc595881e828bde5a9e08ee58287ade9e5bcd070cd6d2a0cd8f0bb4a \
                     size    7518019
 
-compiler.cxx_standard 2017
+compiler.cxx_standard 2011
```

Comment:
Please do not make this change, as 2017 was found to be required for older systems.

Reply:
Fixed thanks.